app/main/appium-method-handler.js

```diff
@@ -232,6 +232,17 @@ export default class AppiumMethodHandler {
       }
 
       if (currentContext !== NATIVE_APP) {
+        const isAndroid = _.toLower(platformName) === 'android';
+        try {
+          // Get the webview offset
+          const el = await this.driver.elementOrNull(
+            isAndroid ? 'xpath' : '-ios class chain',
+            isAndroid ? '//android.webkit.WebView' : '**/XCUIElementTypeWebView'
+          );
+          if (!webViewPosition) {
```

Comment:
sorry, I meant we need to check if `el` is null

Reply:
🤦 , fixed it